src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -16,6 +16,7 @@ foam.CLASS({
 
   imports: [
     'auth',
+    'currentMenu',
```

Comment:
What if someone is using without menus? Like a single Controller app?

Reply:
Right, we only use the currentMenu to fetch the full path so that we're able to translate the title and subtitle. In the case where currentMenu is not set, I'll add a condition where it will default to config$title and config$subtitle.

Reply:
Then, if the currentMenu import is optional, this line should change to 'currentMenu?', otherwise it will generate an error if it is missing. But, does it really even need to know about menus? Couldn't it just translate the title and subtitle that are provided in its config? The .handler.config.browseTitle pattern seems a little too internal implementation specific. Couldn't I instead just use something like "Edit Bank Accounts" as the translation key?

Reply:
Right, that does seem like the better solution in this case. Will update.